shard.lock

```diff
@@ -53,7 +53,7 @@ shards:
     version: 0.1.4
 
   etcd:
-    git: https://github.com/place-labs/crystal-etcd.git
+    git: https://github.com/aca-labs/crystal-etcd.git
```

Comment:
was the revert to aca-labs intentional?

Reply:
nope, i don't know where they're coming potentially some weird name resolution?

Reply:
either way, same version. i think they'll probably revert if the versions are pinned

Reply:
ahh yeah it redirects, all good